### PR TITLE
magic social oauth with popup

### PR DESCRIPTION
**Pull request by intergalacticspacehighway**
# Why
- Enabled login with window popup instead of redirect. This is required for better UX in social connect in onboarding. 
- Magic API is still incredibly slow. Asked them in telegram again to escalate it ASAP

https://user-images.githubusercontent.com/23293248/223726392-70a3f6e9-99fa-4456-bedd-91884bd5aa9c.mov

# How
Locally forked magic oauth and patched it with `loginWithPopup`. Patch package is not possible as they ship it with a minified sdk 😤 

# Test Plan
- Test login with google and apple. Login should happen in window popup instead of redirection.

### CODE REVIEW

File: packages/app/lib/social-logins/index.ts [hunk 1]
```diff
@@ -15,62 +13,35 @@ export const useMagicSocialAuth = () => {
...
-        scope,
-      });
-    } else {
-      return magic.oauth.loginWithPopup({
-        provider: "google",
-        scope,
-        redirectURI: redirectUri,
-      });
-    }
+    return magic.oauth.loginWithPopup({
```
Comment (hirbod) [inline packages/app/lib/social-logins/index.ts:28]: Does loginWithPopup work correctly native?

Comment (intergalacticspacehighway) [reply packages/app/lib/social-logins/index.ts:28]: yeah, it was originally just working on native. This PR makes it work on web.

Comment (intergalacticspacehighway) [reply packages/app/lib/social-logins/index.ts:28]: Test it on dev.showtime.xyz instead of preview url!

**Review by hirbod — APPROVED** [review]
LGTM🔥

**Pull request merged**